portal/views/assessment_engine.py

```diff
@@ -774,8 +778,9 @@ def get_assessments():
 
 @assessment_engine_api.route(
     '/patient/<int:patient_id>/assessment',
-    methods=('PUT',),
+    methods=('OPTIONS', 'PUT')
```

Comment:
@pbugni 
Can we add these additional methods (required for CORS) with the `@crossdomain` decorator?

Reply:
I don't believe so - I think we will need to call out the additional `OPTIONS` method in the `methods` parameter to `route()`